### DISCUSSION TITLE
Formula of model

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1456] [Questions & Problems]
Hello, I have been using PK-SIM to model drug-drug interactions recently. I would like to ask if I can check the formula of the model.

**Comment by StephanSchaller** [comment]
--> Export to MoBi ;-)

**Comment by Yuri05** [comment]
s. also the documentation https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-defining-inhibition-induction-processes

**Reply by WTY-tt** [reply]
Thank you.
